### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by mpoke**
## Type of change

- [x] `Docs`: Adds documentation

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+____
+
+#### For Admin Use
+
+- [ ] Not duplicate issue
+- [ ] Appropriate labels applied
+- [ ] Appropriate contributors tagged
+- [ ] Contributor assigned/self-assigned
+- [ ] Is a spike necessary to map out how the issue should be approached?
```
Comment (smarshall-spitzbart) [inline .github/ISSUE_TEMPLATE/bug-report.md:33]: Do we use spikes in ICS?

**Review by smarshall-spitzbart — APPROVED** [review]
LGTM overall! The `needs-triage` label will automatically add these issues to the project board right?

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+____
+
+#### For Admin Use
+
+- [ ] Not duplicate issue
+- [ ] Appropriate labels applied
+- [ ] Appropriate contributors tagged
+- [ ] Contributor assigned/self-assigned
+- [ ] Is a spike necessary to map out how the issue should be approached?
```
Comment (MSalopek) [inline .github/ISSUE_TEMPLATE/feature-request.md:37]: Sorry, what is a "spike" in this context?

**Review by MSalopek — APPROVED** [review]
Looks good! Thanks for finding the time to do this.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (mpoke) [reply .github/ISSUE_TEMPLATE/bug-report.md:33]: Not sure.

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 2 above)
Comment (mpoke) [reply .github/ISSUE_TEMPLATE/feature-request.md:37]: IIUC,  https://en.wikipedia.org/wiki/Spike_(software_development)

**Pull request merged**